source/patterns/@aws-solutions-constructs/core/test/sqs-helper.test.ts

```diff
@@ -17,6 +17,35 @@ import * as sqs from 'aws-cdk-lib/aws-sqs';
 import * as defaults from '../';
 import '@aws-cdk/assert/jest';
 import { buildDeadLetterQueue, buildQueue } from "../lib/sqs-helper";
+import * as kms from 'aws-cdk-lib/aws-kms';
+import { getResourceLogicalIdFromDescription } from "../";
+
+// --------------------------------------------------------------
+// Test deployment w/ imported encryption key
+// --------------------------------------------------------------
+test('Test deployment w/ encryptionMasterKey set on queueProps', () => {
```

Comment:
I'd noticed that sqs-helper test coverage was missing this case, so I added in to get it to 100% :)